### DISCUSSION TITLE
Any alternatives for ZoomedScene class?

### DISCUSSION THREAD

**Discussion by shuaibw** [3b1b/manim#1642] [Q&A] [answered]
Hi,
I would like to zoom on a particular portion of my scene. Manim previously had a ZoomedScene class, but in the latest version, that has been removed. Are there any alternatives?
![image](https://user-images.githubusercontent.com/57074038/136650196-7eaf47b1-a668-44d1-93de-7a5b7d354078.png)

**Comment by TonyCrane** [comment, marked as answer]
You can just scale `self.camera.frame` to achieve the same effect.
